core/src/main/java/com/linecorp/armeria/common/HttpRequest.java

```diff
@@ -267,6 +267,62 @@ static HttpRequest of(RequestHeaders headers, Publisher<? extends HttpObject> pu
         }
     }
 
+    /**
+     * Creates a new {@link HttpRequestBuilder}.
+     */
+    static HttpRequestBuilder builder() {
+        return new HttpRequestBuilder();
+    }
+
+    /**
+     * Shortcut to create a new {@link HttpRequestBuilder} with OPTIONS method and path.
+     */
+    static HttpRequestBuilder options(String path) {
```

Comment:
I'm somewhat concerned that these methods make `HttpRequest` more crowded. How about moving them to `HttpRequestBuilder`? E.g. `HttpRequest.builder().get("/{foo}")...`